pkg/gameservers/controller.go

```diff
@@ -521,6 +521,7 @@ func (c *Controller) syncGameServerCreatingState(ctx context.Context, gs *agones
 				})
 			}
 		}
+
```

Comment:
nit: while this newline is probably chill, there's no other diff in this file, so maybe minimize the impact and kill it :)